### PR TITLE
Feature/grid mode

### PR DESCRIPTION
**Pull request by pablopareja**
The functionality requested should already be included in this pull request.
I think that only Grid/list button styling when active is missing.
@mbarrenechea could you have a look at this? Thanks!

### CODE REVIEW

**Comment by pablopareja** [comment]
@mbarrenechea Can you have a look at the styles from the last commit?

File: src/components/explore/DatasetListHeader/style.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    margin-top: 12px;
+    margin-left: 6px;
+  }
+
+  .actions {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .c-button {
```
Comment (mbarrenechea) [inline src/components/explore/DatasetListHeader/style.scss]: We talked about this. Try to use the tag instead of the `c-...`.

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Everything looks great but fix this little thing and I will merge

**Comment by pablopareja** [comment]
It should be fixed with this commit

**Pull request merged**